docs(aps): review docs 2025-08-05 (#5375)
scaleway/docs-content: tutorials/configure-plex-s3/index.mdx

- The **Plex Server** application, which is available for Windows, macOS, Linux, and even some NAS devices.
- **Plex clients** that can be either a web-based interface, an application on smart TVs, streaming boxes, or other third-party applications.

<Message type="note">

>   Plex changed it's licensing policy recently and it requires a paid license to stream media. An open-source self-hosted alternative to Plex is [Jellyfin](https://jellyfin.org/)

Reviewer: ```suggestion
  Plex changed its licensing policy recently. A paid license is now required to stream media. An open-source self-hosted alternative to Plex is [Jellyfin](https://jellyfin.org/)
```
